### ISSUE TITLE
Running evcouplings on a cluster without network access

### ISSUE THREAD

**Issue by ld139** [debbiemarkslab/EVcouplings#322]
Hi,

I am planning to run evcouplings on a cluster that does not have network access. Could you please advise on the best approach to prepare the necessary data for running the tool in an offline environment?

Should I download the MMTF files in advance and store them in a folder (e.g., pdb_mmtf_dir in config file )? Alternatively, is it preferable to download the BCIF files or the BCIF.gz compressed files instead?

Any guidance on how to prepare the data files for offline use would be greatly appreciated!

Thanks!

**Comment by thomashopf** [comment]
Yes, I would start with downloading the PDB in .bcif.gz format and reference the directory via `pdb_mmtf_dir` (the name of that parameter is a bit outdated due to the recent phase-out of the MMTF format). 

Also, the input sequence needs to be supplied as a file via the `sequence_file` parameter.

**Comment by ld139** [comment]
> 是的，我将首先下载 .bcif.gz 格式的 PDB，然后通过引用目录`pdb_mmtf_dir`（由于最近 MMTF 格式的逐步淘汰，该参数的名称有点过时）。
> 
> 此外，输入序列需要通过参数作为文件提供`sequence_file`。

Great!

**Comment by ld139** [comment]
i have download all the .bcif.gz files in a folder. but the problem still exist, my command here:

    evcouplings \
    -P output/$protein_dir/$query_protein_name \
    -p $query_protein_name \
    -s $protein_path \
    -b "0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9" \
    -n 5 \
    -y \
    src/single_config_monomer.txt \
the config file and wrong file:
[mcaupo_b0.1.failed.txt](https://github.com/user-attachments/files/18132974/mcaupo_b0.1.failed.txt)
[mcaupo_b0.1_config.txt](https://github.com/user-attachments/files/18132975/mcaupo_b0.1_config.txt)

**Comment by thomashopf** [comment]
Does the folder `/export/home/luod111/af2_dataset/pdb_bcif` contain `2yor.bcif.gz` in exactly this lowercase spelling (e.g. if PDB ID is uppercase, it won't detect the file)?

**Comment by ld139** [comment]
> Does the folder `/export/home/luod111/af2_dataset/pdb_bcif` contain `2yor.bcif.gz` in exactly this lowercase spelling (e.g. if PDB ID is uppercase, it won't detect the file)?

yes, i'm sure `/export/home/luod111/af2_dataset/pdb_bcif` folder contiain `2yor.bcif.gz` file!
![1734703215119](https://github.com/user-attachments/assets/bfd57797-5677-4eef-8a3a-69ec328aeadf)

**Comment by thomashopf** [comment]
Are you on the latest version from the `develop` branch? If not, please try with this version.

I just double-checked and the updated code to load .bcif.gz files from a directory is not yet on `master` or the pip-installable version
